### PR TITLE
optional fix for https port 5284 error and quick install guide updates

### PR DESCRIPTION
**Pull request by crouchingtigerhiddenadam**
https://community.jitsi.org/t/no-certificate-present-in-ssl-tls-configuration-for-https-port-5284/47836/7

For me, this fix prevents the `No certificate present in SSL/TLS configuration for https port 5284` error which appears to be linked to prosody shutting down on a few of machines I've tested running Debian Buster 10.

### CODE REVIEW

File: doc/debian/jitsi-meet-prosody/prosody.cfg.lua-jvb.example [hunk 1]
```diff
@@ -13,6 +13,7 @@ turncredentials = {
 
 cross_domain_bosh = false;
 consider_bosh_secure = true;
+-- https_ports = { }; -- Remove this line to prevent listening on port 5284
```
Comment (saghul) [inline doc/debian/jitsi-meet-prosody/prosody.cfg.lua-jvb.example:16]: I wonder if we should just have it uncommented, thus disabled by default. Thoughts @damencho ?

File: doc/quick-install.md [hunk 2]
```diff
@@ -114,13 +114,13 @@ If this all worked, then congratulations!  You have an operational Jitsi confere
...
 ```sh
-apt-get -y install jigasi
+sudo apt install jigasi
 ```
 or
 
 ```sh
 wget https://download.jitsi.org/unstable/jigasi_1.0-107_amd64.deb
-dpkg -i jigasi_1.0-107_amd64.deb
+sudo dpkg -i jigasi_1.0-107_amd64.deb
```
Comment (saghul) [inline doc/quick-install.md:123]: Can you actually delete this "or" ection entirely? Jigasi is now on the staable repo so there is no need to take it from the unstable one.

Comment (crouchingtigerhiddenadam) [reply doc/quick-install.md:123]: I can change that. I can run through the instructions again using the stable branch and test it, or change directly to https://download.jitsi.org/stable/jigasi_1.1-107-g6928850-1_amd64.deb?

Comment (saghul) [reply doc/quick-install.md]: No, I mean remve everything after the "or". Just leave the existing `apt install jigasi` that's all there is to it now.

Comment (crouchingtigerhiddenadam) [reply doc/quick-install.md]: I've applied the change to 77e0709: wget/dpkg removed from jagasi instructions

Comment (saghul) [reply doc/quick-install.md]: Looking good!

File: doc/debian/jitsi-meet-prosody/prosody.cfg.lua-jvb.example (hunk 1 above)
Comment (damencho) [reply doc/debian/jitsi-meet-prosody/prosody.cfg.lua-jvb.example:16]: Well, maybe yeah ... Anyway, we started modifying global configs ... The dangerous thing is that if somebody had prosody already with some config we can break it ...

Comment (saghul) [reply doc/debian/jitsi-meet-prosody/prosody.cfg.lua-jvb.example:16]: Let's leave the comment then.

**Pull request merged**